core/src/main/java/com/netflix/conductor/core/orchestration/ExecutionDAOFacade.java

```diff
@@ -159,9 +159,8 @@ public Workflow getWorkflowById(String workflowId, boolean includeTasks) {
             LOGGER.debug("Workflow {} not found in executionDAO, checking indexDAO", workflowId);
             String json = indexDAO.get(workflowId, RAW_JSON_FIELD);
             if (json == null) {
-                String errorMsg = String.format("No such workflow found by id: %s", workflowId);
```

Comment:
this change will affect many places. as getWorkflowById() is used in many places all those callers might be coded to expect exception if workflow does not present in db. suddenly if you return null, all the other places need to be tested. hence its better not to change here. let it throw exception as usual.

Reply:
done